app/Http/Controllers/Admin/RoomTypeController.php

```diff
@@ -21,13 +36,10 @@ class RoomTypeController extends AdminBaseController
      */
     public function index()
     {
-        $columns = [
-            'id',
-            'name',
-            'quality',
-            'image',
-        ];
-        $roomTypes = RoomType::select($columns)->paginate(20);
+        $roomTypes = RoomType::select($this->columns)->paginate(20);
```

Comment:
it is not a good way. columns to select is difference between action, not same. Define for each function, not property of classs